addons/littlebird_hud/fn_showGreaseDot.sqf

```diff
@@ -0,0 +1,9 @@
+params ["_vehicle", "_enable"];
+
+if (_vehicle isKindOf "RHS_MELB_AH6M") then {
+        private _status = [0, 1] select _enable;
+        _vehicle animateSource ["Addcrosshair", _status, true];
+    } else {
+        private _status = ["", "a3\air_f\heli_light_01\data\heli_light_01_dot_ca.paa"] select _enable;
```

Comment:
Btw, an alternate way of writing this that's closer to what other languages use (as a ternary operator) is
```sqf
private _status = if (_enable) then { "" } else { "a3\air_f\heli_light_01\data\heli_light_01_dot_ca.paa" };
```
Either way is fine, though.